packages/flutter_tools/templates/app/android-kotlin.tmpl/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.2.71'
+    ext.kotlin_version = '1.3.0'
```

Comment:
I think this might break plugins for some people - maybe needs to be announced as a breaking change?

Reply:
Might not be worth the churn in that case -- and better to just wait until we can bump the Gradle version to `5.y.z`...

Reply:
One more reason to migrate to AARs. This will be fixed automatically.